starters/gatsby-starter-theme/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "license": "0BSD",
   "dependencies": {
-    "gatsby": "^2.24.41",
+    "gatsby": "^2.24.42",
```

Comment:
This project has no lock file and the `^` should tell `npm` and `yarn` to install the latest known patch version. In other words, there's no real need to bump the patch version for every time we update the gatsby package. Please undo this change.

Reply:
@pvdz Sadly the project has a lock file and it contains `2.24.41` in it: https://github.com/gatsbyjs/gatsby/blob/master/starters/gatsby-starter-theme/package-lock.json